### PR TITLE
Boarding submitted modal when submitting a boarding

### LINKED ISSUE #191: Boarding Submitted modal when submitting a boarding
After the "Are you sure you want to submit the boarding?" dialog, when a user clicks submit, they are brought back to the home page. This is correct, however they should get a dialog that says "Boarding Submitted!" and they have to click "OK" to continue. See screenshot of how it looks on Android:

Screen Shot 2020-08-11 at 1 44 38 PM

### PR DESCRIPTION
**Pull request by akaktus**
[-] Added alert after submitting boarding.
Fixes #191

### CODE REVIEW

File: o-fish-ios/Views/PatrolBoat/PatrolBoatView.swift [hunk 1]
```diff
@@ -18,6 +18,7 @@ struct PatrolBoatView: View {
     @State private var isActiveRootFromSearchView = false
     @State private var isActiveRootFromShowingPatrolSummaryView = false
     @State private var resetLocation = {}
+    @State private var showingBoardingSubmitted = false
```
Comment (kojiba) [inline o-fish-ios/Views/PatrolBoat/PatrolBoatView.swift:21]: we can show this alert from any screen, what is the purpose to add this state/bindings arguments flow?

File: o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift [hunk 2]
```diff
@@ -96,13 +98,15 @@ struct ReportNavigationRootView: View {
         print("Saving report in Realm")
         report.save()
         rootIsActive.toggle()
+        showingSubmitted = true
```
Comment (kojiba) [inline o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift:101]: just show alert here and not complicate the views and previews

File: o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift [hunk 3]
```diff
@@ -95,6 +98,9 @@ struct ReportNavigationRootView: View {
     private func saveAlertClicked() {
         print("Saving report in Realm")
         report.save()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```
Comment (kojiba) [inline o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift:101]: timer looks a bit hacky

Comment (akaktus) [reply o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift:101]: It's hack, because swiftUI doesn't have possibility show alert one after another.

**Comment by akaktus** [comment]
Alert is shown on the PatrolBoatView, according design.

**Pull request merged**